src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -344,6 +368,28 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
             if (controllerData.xrController) {
                 controllerGlobalPosition = controllerData.xrController.pointer.position;
                 controllerData.xrController.getWorldPointerRayToRef(controllerData.tmpRay);
+                const hand = controllerData.xrController.inputSource.hand;
+                if(hand) {
+                    const xrIndexTip = hand.get("index-finger-tip");
+                    if(xrIndexTip) {
+                        let indexTipPose = _xrFrame.getJointPose!(xrIndexTip, this._xrSessionManager.referenceSpace);
+                        if (indexTipPose && indexTipPose.transform) {
+                            let zAxisMultiplier = 1;
+                            if(!this._scene.useRightHandedSystem)

```

Comment:
This can also be one-lined as the following if you want:
let zAxisMultiplier = this._scene.useRightHandedSystem ? 1 : -1;